src/models/environment.ts

```diff
@@ -25,6 +25,7 @@ const schema = Joi.object().keys({
   locale: localeSchema.required().example('en'),
   expiresAt: Joi.date().allow(null).required().example(null),
   createdAt: Joi.date().required().example('2019-12-31T15:23Z'),
+  defaultGraphRange: Joi.string().allow(null).required().example('30d'),
```

Comment:
is `allow(null)` needed? It's not reflected on type.

Reply:
Updated